frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -120,9 +121,9 @@ function MethodologyTab() {
                 <p>
                   For a description of some of the gaps in COVID-19 data, please
                   see the{" "}
-                  <a href={EXPLORE_DATA_PAGE_WHAT_DATA_ARE_MISSING_LINK}>
+                  <Link to={EXPLORE_DATA_PAGE_WHAT_DATA_ARE_MISSING_LINK}>
```

Comment:
This one didn't work for me, it brought me to the middle of the exploredata page

Reply:
weird, it's broken for me now; was working before. I wondered why we are linking back to this section at all, I made another issue for this but to make i feel like the INFO should live on the methodology and the DATA should live on the tracker...

Reply:
Implemented some min heights for the cards; that should be in place anyways because cumulative layout shift is a) really annoying to the user and b) penalized by Google ranking. By adding min heights to the objects that take a while to load you can minimuze how much the page shifts around. This seems to have fixed / helped a lot with letting the link scroll to the info section properly